File: typescript-client-sdk/src/sdk/models/operations/deletewebhook.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class DeleteWebhookRequestBody extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=accessToken" })
  accessToken?: string;

  @SpeakeasyMetadata({ data: "json, name=webhookId" })
  webhookId?: string;
}


export class DeleteWebhookRequest extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "request, media_type=application/json" })
  request?: DeleteWebhookRequestBody;
}


export class DeleteWebhookResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;
}

File: go-client-sdk/pkg/models/operations/deletewebhook.go
package operations

type DeleteWebhookRequestBody struct {
	AccessToken *string `json:"accessToken,omitempty"`
	WebhookID   *string `json:"webhookId,omitempty"`
}

type DeleteWebhookRequest struct {
	Request *DeleteWebhookRequestBody `request:"mediaType=application/json"`
}

type DeleteWebhookResponse struct {
	ContentType string
	StatusCode  int64
}
